<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Ameet" failures="10" tests="11" name="Test" time="160.677" errors="0" timestamp="2025-07-11T11:38:25 IST">
  <testcase classname="testcases.RestAssured_TestCases" name="loginWithSeleniumAndGetCookie" time="113.228">
    <failure type="org.openqa.selenium.WebDriverException" message="Timed out waiting for driver server to stop.
Build info: version: &amp;apos;4.10.0&amp;apos;, revision: &amp;apos;c14d967899&amp;apos;
System info: os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17.0.7&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [b424b4bcee166c6a31bc2cce8abeb2a4, quit {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 138.0.7204.97, chrome: {chromedriverVersion: 138.0.7204.94 (545025748884..., userDataDir: C:\Users\AMEETK~1\AppData\L...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:49558}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:49558/devtoo..., se:cdpVersion: 138.0.7204.97, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: b424b4bcee166c6a31bc2cce8abeb2a4">
      <![CDATA[org.openqa.selenium.WebDriverException: Timed out waiting for driver server to stop.
Build info: version: '4.10.0', revision: 'c14d967899'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.7'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [b424b4bcee166c6a31bc2cce8abeb2a4, quit {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 138.0.7204.97, chrome: {chromedriverVersion: 138.0.7204.94 (545025748884..., userDataDir: C:\Users\AMEETK~1\AppData\L...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:49558}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:49558/devtoo..., se:cdpVersion: 138.0.7204.97, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: b424b4bcee166c6a31bc2cce8abeb2a4
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:161)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:531)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:590)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:594)
at org.openqa.selenium.remote.RemoteWebDriver.quit(RemoteWebDriver.java:446)
at org.openqa.selenium.chromium.ChromiumDriver.quit(ChromiumDriver.java:341)
at testcases.RestAssured_TestCases.loginWithSeleniumAndGetCookie(RestAssured_TestCases.java:88)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.util.concurrent.ExecutionException: org.openqa.selenium.TimeoutException: Process timed out after waiting for 20000 ms.
Build info: version: '4.10.0', revision: 'c14d967899'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.7'
Driver info: driver.version: unknown
at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396)
at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2096)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:157)
... 21 more
Caused by: org.openqa.selenium.TimeoutException: Process timed out after waiting for 20000 ms.
Build info: version: '4.10.0', revision: 'c14d967899'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.7'
Driver info: driver.version: unknown
at org.openqa.selenium.os.OsProcess.waitFor(OsProcess.java:183)
at org.openqa.selenium.os.CommandLine.waitFor(CommandLine.java:117)
at org.openqa.selenium.remote.service.DriverCommandExecutor.lambda$execute$2(DriverCommandExecutor.java:148)
at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
... 3 more
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- loginWithSeleniumAndGetCookie -->
  <testcase classname="testcases.RestAssured_TestCases" name="GetLogin" time="0.04">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;io.restassured.response.Response.getBody()&amp;quot; because &amp;quot;response&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "io.restassured.response.Response.getBody()" because "response" is null
at rest.CustomResponse.<init>(CustomResponse.java:64)
at rest.ApiUtil.GetLogin(ApiUtil.java:127)
at testcases.RestAssured_TestCases.GetLogin(RestAssured_TestCases.java:104)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- GetLogin -->
  <testcase classname="testcases.RestAssured_TestCases" name="GetEmpActionSummary" time="0.008">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;io.restassured.response.Response.getBody()&amp;quot; because &amp;quot;response&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "io.restassured.response.Response.getBody()" because "response" is null
at rest.CustomResponse.<init>(CustomResponse.java:64)
at rest.ApiUtil.GetEmpActionSummary(ApiUtil.java:155)
at testcases.RestAssured_TestCases.GetEmpActionSummary(RestAssured_TestCases.java:132)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- GetEmpActionSummary -->
  <testcase classname="testcases.RestAssured_TestCases" name="GetDashboardShortcut" time="0.001">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;io.restassured.response.Response.getBody()&amp;quot; because &amp;quot;response&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "io.restassured.response.Response.getBody()" because "response" is null
at rest.CustomResponse.<init>(CustomResponse.java:98)
at rest.ApiUtil.GetDashboardShortcut(ApiUtil.java:191)
at testcases.RestAssured_TestCases.GetDashboardShortcut(RestAssured_TestCases.java:156)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- GetDashboardShortcut -->
  <testcase classname="testcases.RestAssured_TestCases" name="GetEmpLeaveInfo" time="0.02">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;io.restassured.response.Response.getBody()&amp;quot; because &amp;quot;response&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "io.restassured.response.Response.getBody()" because "response" is null
at rest.CustomResponse.<init>(CustomResponse.java:64)
at rest.ApiUtil.GetEmpLeaveInfo(ApiUtil.java:219)
at testcases.RestAssured_TestCases.GetEmpLeaveInfo(RestAssured_TestCases.java:201)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- GetEmpLeaveInfo -->
  <testcase classname="testcases.RestAssured_TestCases" name="GetEmpSubunit" time="0.001">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;io.restassured.response.Response.getBody()&amp;quot; because &amp;quot;response&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "io.restassured.response.Response.getBody()" because "response" is null
at rest.CustomResponse.<init>(CustomResponse.java:130)
at rest.ApiUtil.GetEmpSubunit(ApiUtil.java:249)
at testcases.RestAssured_TestCases.GetEmpSubunit(RestAssured_TestCases.java:224)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- GetEmpSubunit -->
  <testcase classname="testcases.RestAssured_TestCases" name="PutEmpName" time="26.887">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;io.restassured.response.Response.getBody()&amp;quot; because &amp;quot;response&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "io.restassured.response.Response.getBody()" because "response" is null
at rest.CustomResponse.<init>(CustomResponse.java:161)
at rest.ApiUtil.PutEmpName(ApiUtil.java:285)
at testcases.RestAssured_TestCases.PutEmpName(RestAssured_TestCases.java:263)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- PutEmpName -->
  <testcase classname="testcases.RestAssured_TestCases" name="PostEmpStatus" time="0.279">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;io.restassured.response.Response.getBody()&amp;quot; because &amp;quot;response&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "io.restassured.response.Response.getBody()" because "response" is null
at rest.CustomResponse.<init>(CustomResponse.java:190)
at rest.ApiUtil.PostEmpStatus(ApiUtil.java:319)
at testcases.RestAssured_TestCases.PostEmpStatus(RestAssured_TestCases.java:298)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- PostEmpStatus -->
  <testcase classname="testcases.RestAssured_TestCases" name="DeleteEmp" time="2.306">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;io.restassured.response.Response.getBody()&amp;quot; because &amp;quot;response&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "io.restassured.response.Response.getBody()" because "response" is null
at rest.CustomResponse.<init>(CustomResponse.java:64)
at rest.ApiUtil.DeleteEmp(ApiUtil.java:348)
at testcases.RestAssured_TestCases.DeleteEmp(RestAssured_TestCases.java:341)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- DeleteEmp -->
  <testcase classname="testcases.RestAssured_TestCases" name="PostEmpName" time="1.205">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;io.restassured.response.Response.getBody()&amp;quot; because &amp;quot;response&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "io.restassured.response.Response.getBody()" because "response" is null
at rest.CustomResponse.<init>(CustomResponse.java:190)
at rest.ApiUtil.PostEmpName(ApiUtil.java:417)
at testcases.RestAssured_TestCases.PostEmpName(RestAssured_TestCases.java:411)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- PostEmpName -->
  <testcase classname="testcases.RestAssured_TestCases" name="PutEmpStatus" time="0.0">
    <skipped/>
  </testcase> <!-- PutEmpStatus -->
</testsuite> <!-- Test -->
